### ISSUE TITLE
Signed actions

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#30266]
**Is your feature request related to a problem? Please describe.**
Let's say you have an application and you want to embed some Metabase data app into it (there were a few interesting use cases commented on the webinar we just had where users use data apps to create simple logic that they just don't want to make on their apps). Right now, signed embedding doesn't work with actions, as it needs you to be authenticated, but it would be a really nice addition to make actions to work in that context, how?

well, in the same way you can embed a dashboard with signed parameters so users can't modify those (signed + hidden), we could populate the parameters that actions need in the same way, so users would be able to embed data apps in their own apps and users would just need to populate the fields that are left open (it's the same logic as we do with filters in signed embedding: we have filters that are disabled, others that are open and others that are locked and get hidden)

**Describe the solution you'd like**
Allow data apps in signed embedding contexts, with the possibility of passing values that come in the JWT to the parameters of the action/s

**Describe alternatives you've considered**
None

**How important is this feature to you?**
Just came up with this idea, no one has asked this yet

**Additional context**
NA

**Comment by vbenedetti** [comment]
Luis, could you add a link here to the webminar you mentioned so that we can check on the relevant comments?

**Comment by paoliniluis** [comment]
yup! @vbenedetti this one https://www.metabase.com/events/metabase-46-ready-set-action
